ansible/adrl.yml

```diff
@@ -1,13 +1,12 @@
 - name: ADRL on CentOS
   hosts: all
-  remote_user: vagrant
+  remote_user: "{{ lookup('env', 'REMOTE_USER') }}"
```

Comment:
Elsewhere we use the variable ansible_ssh_user, please use that here for consistency.